pages/1.11/monitoring/logging/logging-api/index.md

```diff
@@ -11,5 +11,81 @@ enterprise: false
 <!-- This source repo for this topic is https://github.com/dcos/dcos-docs -->
 
 
-There are 2 versions of the logging API
+The Logging API exposes node, component, and container (task) logs.
 
+- The Logging API is backed by the [DC/OS Log component](/1.11/overview/architecture/components/#dcos-log), which runs on all nodes in the cluster.
+
+- For more information about using the Logging API, see [Logging](/1.11/monitoring/logging/index.md).
+
+- For usage examples, see [Logging API Examples](/1.11/monitoring/logging/logging-api-examples/index.md).
+
+# Compatibility
+
+The Logging API has been updated significantly for DC/OS 1.11.
+
+
+## Logging features new for DC/OS 1.11
+
+In versions of DC/OS prior to 1.11, task logs were available via [files API](http://mesos.apache.org/documentation/latest/endpoints/#files-1). Now the user can leverage the consolidated API *for both component and task logs*.
+
+## Notes on Previous Versions
+
+The Logging API was first added in DC/OS 1.9.0. Prior to DC/OS 1.9.0, all node, component, and container logs were managed by Logrotate.
```

Comment:
as for managed by logrotate, it is not quite right.
node and component logs were (and are ) stored in journald binary log. We just did not have any APIs to retrieve them. So before 1.9 our CLI would ssh into a node and run `journalctl` linux command to get the logs.

Reply:
Cool, I simplified it and yanked the logrotate language I got from the old Logging API v1 doc.